server/task.go

```diff
@@ -461,8 +461,9 @@ func (s *Server) changeTaskStatusWithPatch(ctx context.Context, task *api.Task,
 	}
 
 	// If create database or schema update task completes, we sync the corresponding instance schema immediately.
-	if (taskPatched.Type == api.TaskDatabaseCreate || taskPatched.Type == api.TaskDatabaseSchemaUpdate) &&
-		taskPatched.Status == api.TaskDone {
+	// On gh-ost tasks, sync instance schema immediately.
+	if ((taskPatched.Type == api.TaskDatabaseCreate || taskPatched.Type == api.TaskDatabaseSchemaUpdate) && taskPatched.Status == api.TaskDone) ||
```

Comment:
Or should we add these two task types to the first clause (taskPatched.Type == api.TaskDatabaseCreate || taskPatched.Type == api.TaskDatabaseSchemaUpdate)?

And shall we do it on ghost cutover task only without sync?

Reply:
Done